src/index.js

```diff
@@ -38,7 +38,7 @@ class AuthGoogleXoPlugin {
     xo.registerPassportStrategy(new Strategy(this._conf, async (accessToken, refreshToken, profile, done) => {
       try {
         console.log(profile)
-        done(null, await xo.registerUser('google', profile.displayName))
+        done(null, await xo.registerUser('google', profile.emails[0].value))
```

Comment:
I believe it should be a fallback to make sure we don't break people for whom it currently works :)